src/ua/sumdu/j2se/Potapenko/tasks/ArrayTaskList.java

```diff
@@ -0,0 +1,76 @@
+package ua.sumdu.j2se.Potapenko.tasks;
+
+public class ArrayTaskList{
+    Task[] task = new Task[0];
```

Comment:
Also, memory-wise it's better to add some DEFAULT_CAPACITY of, let's say, 100 elements so you won't have to recreate an array each time user adds a new task